kong/pdk/init.lua

```diff
@@ -37,25 +37,6 @@
 -- end
 
 
----
```

Comment:
The top-comment of this file needs to be modified - after this change, the PDK is no longer "Semantically Versioned".